Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * Control Flow Graph: Fix incorrectly reported unreachable code.
```

Comment:
Do you in principal credit bug discoverers or actually no one? If the former is the case, I'm the discoverer ;)

Reply:
In any case, I'm amazed what in-depth discussion resulted from my findings. Really cool.

Reply:
I see you don't give any credits in the Changelog. That's okay, we dont want to break your consistency :D

Reply:
Sorry, we only attribute "critical bugs", i.e. those that make it into the [bug list](https://docs.soliditylang.org/en/latest/bugs.html) and where we write blog posts about. We also don't credit who implemented which feature :)

Reply:
Sure! Thanks for clarifying. I'm happy as it is ☺️